helper_scripts/main_install_script.sh

```diff
@@ -48,8 +48,8 @@ readonly TEMPLATE_DIR="${INSTALLER_PWD}/buildout_templates"
 
 # --------- substitute PYTHON[3]_{URL,DIR,TB} variables ...
 . "${HSCRIPTS_DIR}/python_versions.sh"
-readonly VIRTUALENV_TB=virtualenv-16.7.8.tar.gz
-readonly VIRTUALENV_DIR=pypa-virtualenv-c85afa5
+readonly VIRTUALENV_TB=virtualenv-20.0.28.tar.gz
+readonly VIRTUALENV_DIR=virtualenv-20.0.28
```

Comment:
I would say, stick to virtualenv 16. @jensens bumped it to 20, but @tkimnguyen reverted this, because it gave problems. See also https://community.plone.org/t/merged-prs-on-unified-installer/12817

But maybe in combination with all the other changes in this PR it works.